Orange/widgets/data/owselectcolumns.py

```diff
@@ -263,53 +100,24 @@ class Outputs:
 
     def __init__(self):
         super().__init__()
+        VariablesFilter().__init__()
```

Comment:
Don't call a constructor of a non-base class!

Reply:
Well, thanks, forgot to remove it. Firstly I wanted to implement it in a different way.